### DISCUSSION TITLE
Is there any way to see the Made For You playlists?

### DISCUSSION THREAD

**Discussion by soriac** [kraxarn/spotify-qt#150] [Q&A] [answered]
I got everything running under windows with librespot and it works great for listening to my playlists, but I really enjoy the Daily Mixes, Discover Weekly and Release radar (basically the whole Made for You section).

Am I being blind or is there no way to access these playlists? I searched around issues and discussions too but found nothing.

**Comment by kraxarn** [comment, marked as answer]
Afaik, there's no way to get them from the API, but they are treated like normal playlists, so it's possible that they could be found using the search feature, which might actually be worth looking into.

The workaround I've been using is to simply follow the playlists, which makes them show up in the normal list over playlists, as shown in the screenshot from the readme.

**Reply by soriac** [reply]
Oh! I didn't know you could follow those playlists. Thank you very much! That works great.
